docs/source/app-auth-update.rst

```diff
@@ -0,0 +1,42 @@
+Updating an App's credentials
+=============================
+
+Overview
+--------
+
+You may need to change an App's credentials due to internal rotation policies or otherwise. The StreamAlert CLI allows you to easily update App credentials.
+to aid in this process, the CLI also give you the ability to list currently configured Apps.
+
+Listing Apps
+------------
+
+To list currently configured StreamAlert Apps (grouped by cluster), use the CLI command:
```

Comment:
`To list your currently deployed StreamAlert Apps:`

Reply:
^^ That's not accurate, since just because it's in the config doesn't mean it's deployed